packages/grid/enclave/enclave-development.md

```diff
@@ -0,0 +1,63 @@
+# Enclave Development
+
+Note: Attestation currently works only in Linux x64
+
+## Building Attestion
+
+```sh
+cd packages/grid/enclave && \
+docker build -f attesation.dockerfile  . -t attestation:0.1
+```
+
+## For CPU Attestation
+
+```sh
+docker run -it --rm --privileged \
+  -v /sys/kernel/security:/sys/kernel/security \
+  -v /dev/tpmrm0:/dev/tpmrm0 attest:0.1 /bin/bash
```

Comment:
Change tag here from `attest:0.1` to `attestation:0.1`

Reply:
Modified 👍